README.md

```diff
@@ -0,0 +1,40 @@
+# Workspace client
+
+This is a client for workspace REST API and workspace master JSON-RPC API interactions.
+
+## Examples
+
+### REST API client
+
+```typescript
+import { Rest } from 'workspace-client';
```

Comment:
Should the main class be Workspace as it's a workspace client library ?
or WorkspaceClient ?

Reply:
This library is divided into two namespaces: `Rest` holds workspace REST API interactions and `JsonRpc` is for workspace master JSON-RPC API interaction. If you want to import everything from the library you could do
```
import * as WorkspaceClient from 'workspace-client';
```

Reply:
Yes but from client's perspective as I wrote later

I would prefer to use a 'more typed class at top level" and not only import a package
so something like WorkspaceClient.getRest() / WorkspaceClient.getJsonRPC(entrypoint)
It's only a comment.

So it's clear to me else with only `Rest` it's not clear to me that this is import is belonging to the workspace when I'm seing it in the code.

to sumup: package name is workspace client so I expect entry point of the library to contains something related/close to workspace as well.